.gitignore

```diff
@@ -1 +1,6 @@
 node_modules
+.env
```

Comment:
![kody code-review](https://img.shields.io/badge/kody-code--review-brightgreen)

```javascript
# Environment variables - never commit sensitive data
*.env*

# Specific environment files
.env
.env.local
.env.development.local
.env.test.local
.env.production.local

# Template file should be committed
!.env.example
```
Consider adding *.env* pattern to catch any other potential environment files that might be added in the future, and add a comment explaining why these files are ignored



<sub>Talk to Kody by mentioning @kody</sub>

<sub>Was this suggestion helpful? React with 👍 or 👎 to help Kody learn from this interaction.</sub><!-- kody-codereview -->&#8203;
&#8203;